Angular/src/app/sidebar/sidebar.component.html

```diff
@@ -6,9 +6,30 @@
       </a>
     </li>
     <li>
-        <a routerLink="language-add">
-          <i class="material-icons" title="Add New Language">add_circle</i>
-        </a>
-      </li>
+      <a routerLink="language-add">
+        <i class="material-icons" title="Add New Language">add_circle</i>
+      </a>
+    </li>
+  </ul>
+  <ul class="change-ui-lang-button">
```

Comment:
I was initially thinking of having a separate component html. The upside I was thinking of would be that we could always have the languages listed out, so it's more clear where to select the different translations.

Reply:
@jakobh7 Oh, I think this is a language add for end users for adding a paradigms and words